neoscore/core/path.py

```diff
@@ -72,17 +72,31 @@ def straight_line(
         start: PointDef,
         parent: Optional[PositionedObject],
         end: PointDef,
+        end_parent: Optional[PositionedObject] = None,
         brush: Optional[BrushDef] = None,
         pen: Optional[PenDef] = None,
     ) -> Path:
         """Convenience for drawing a single straight line.
 
-        ``end`` is measured relative to the starting point.
+        Args:
+            start: The position of the center of the arrow line's start
+            parent: A parent object
+            end: The position of the end of the line, relative to ``end_parent``
+                if provided or ``start``
+            end_parent: An optional parent for the end point.
+            brush: The brush to fill shapes with.
+            pen: The pen to draw outlines with. Defaults to no pen.
+
+        Note that ``end_parent`` is only used for initially drawing the
+        path. If ``end_parent`` moves relative to the path after
+        creation, the path shape will not be automatically updated.
```

Comment:
this bit was true for `arrow`, but since the parent is actually passed into the path element it's not true here, so this can be removed